src/core/components.py

```diff
@@ -165,6 +167,30 @@ def get_validators(self):
             self.populate.get_validators()
 
 
+class DateTime(Input):
+    __slots__ = [
+        "datetime_type",
+    ]
+
+    def get_datetime_type(self, datetime_type):
+        validate = self.validate_datetime(datetime_type)
+        if validate:
```

Comment:
Is the check here needed?
```python
self.validate_datetime(datetime_type)
return datetime_type
```
won't return if the InvalidArguments exception is raised, no? or is this more for verbosity / code clarity?

Reply:
yes, specificity

Reply:
can remove, but follows pattern implemented elsewhere (selection component)